test/parallel/test-crypto-dh.js

```diff
@@ -307,6 +308,30 @@ if (availableCurves.has('prime256v1') && availableCurves.has('secp256k1')) {
   });
 }
 
+// Use of invalid keys was not cleaning up ERR stack, and was causing
+// unexpected failure in subsequent signing operations.
+if (availableCurves.has('prime256v1') && availableHashes.has('sha256')) {
+  const curve = crypto.createECDH('prime256v1');
+  const invalidKey = Buffer.alloc(65);
+  invalidKey.fill('\0');
+  curve.generateKeys();
+  try {
+    curve.computeSecret(invalidKey);
+    assert(false, 'key generation should have failed');
```

Comment:
But the test case doesn't have to be in the catch block, or am I just missing something?
```
assert.throws(() => curve.computeSecret(invalidKey));
const ecPrivateKey = ...;
assert.doesNotThrow(() => crypto.createSign('SHA256').sign(ecPrivateKey));
```